app/src/main/java/com/github/onlynotesswent/ui/overview/editnote/EditNote.kt

```diff
@@ -126,6 +127,13 @@ fun EditNoteScreen(
                     courseName = courseName,
                     courseYear = courseYear,
                     noteViewModel = noteViewModel)
+                // todo Additional check might be useless if checking is done before
+              } else if (note != null &&
+                  (note!!.visibility == Visibility.PUBLIC ||
```

Comment:
Again, shouldn't we also check for the public note owner to be different from the current user ?

Reply:
Same comment as with folder, and same change made

Reply:
Same as FolderContent comment